django/thunderstore/api/cyberstorm/views/user.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.http import Http404, HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.cyberstorm.serializers import CyberstormUserSerializer
+
+User = get_user_model()
+
+
+class UserAPIView(APIView):
```

Comment:
This whole view should probably just extend the `RetrieveAPIView` and skip implementing anything custom. See: 
- https://www.django-rest-framework.org/api-guide/generic-views/#retrieveapiview
- https://www.django-rest-framework.org/api-guide/generic-views/#genericapiview

Reply:
I've removed user endpoint, since it only returns the username it's being called with. I'll change the team to use RetrieveAPIView though.